client/src/components/dashboard/Mentorship/UserCard.js

```diff
@@ -0,0 +1,122 @@
+import ChatIconPopup from '../common/ChatIconPopup';
+import { hoverTransition } from '../../../styles/style-utils';
+import { isGitterUser } from '../../../actions/user';
+import propTypes from 'prop-types';
+import React from 'react';
+import styled from 'styled-components';
+
+const Card = styled.div`
+  width: 45vw !important;
+`;
+
+const ChatIcon = styled.i`
+  ${ hoverTransition() }
+  font-size: 20px !important;
```

Comment:
@smiller171 all related to `styled-components` and overriding `semantic-ui` styles